.github/workflows/lerna-publish.yml

```diff
@@ -3,31 +3,68 @@ name: Publish NPM Packages
 on:
   workflow_dispatch:
     inputs:
-      branch:
-        description: 'branch to release from'
+      ref:
+        description: 'commit/tag/branch to release from'
```

Comment:
when we feel confident in this publish flow we'll add a `on: release` trigger so it goes automatically once the release-pr is merged and the release-please github action has created tags